src/toil/test/wdl/standard_library/ceil.wdl

```diff
@@ -0,0 +1,16 @@
+workflow ceilWorkflow {
+  Float num
+  call get_ceil { input: num=ceil(num) }
```

Comment:
I wonder if the `ceil` call should be inside `get_ceil`

Reply:
Testing it inside of the call might even be a different test and we could include both.  I think I'll add it as a separate test this time, but as you add in the future, really only one should be necessary since we want to test the function and not necessarily permutations of the syntax.

Cromwell returns a 12.0 (float) in this case and toil returns a 12 (int), which means that they're not really obeying their spec it seems: https://github.com/openwdl/wdl/blob/main/versions/development/SPEC.md#int-floorfloat-int-ceilfloat-and-int-roundfloat and I imagine we'll find other cases where this occurs as well.  We'll also see the holes in toil as well going through this and how large those holes are.

Reply:
I've been testing with Cromwell and it seems like Cromwell's `ceil` does return an Int. In this case, I think Cromwell's behavior (returning 12.0) is actually correct- we're expecting a Float `num` output. I don't know if it's worth it to investigate how Toil does type casting and make it behave exactly like Cromwell. It doesn't matter here, but I suppose there are instances where typing is important.